src/tutorials/region-migration.md

```diff
@@ -0,0 +1,55 @@
+# Region migration
+
+Platform.sh is available in a number of different *Regions*.  Each region is a self-contained copy of Platform.sh in a single datacenter.  When you first create a project you can specify what region it should be in.
+
+There is no automated way to migrate a project from one region to another after it is created.  However, the process to do so manually is fairly straightforward.
+
+## Why migrate regions?
+
+* Different datacenters are located in different geographic areas, and you may want to keep your site physically close to the bulk of your user base for reduced latency.
+* Only selected regions offer European Sovereign hosting.  If you created a project in a non-Sovereign region you may need to migrate to a Sovereign region.
+* Some regions are running older versions of the Platform.sh codebase that offers fewer features.  In particular, the US-1 and EU-1 regions do not offer XL and 2XL plans, self-terminating builds in case of a build process that runs too long, or distributing environments across different grid hosts.  If you are on one of those regions and desire those features you will need to migrate to the newer US-2 or EU-2 regions.
+
+## Migration process
+
+### 0. Preparation
+
+* Plan a timeframe in which to handle the migration.  You will want to avoid developing any new code during that period (as your Git repository will change) and be prepared for a brief site outage when you migrate.  You are essentially relaunching the site, just with the same host as previously.  Plan accordingly.
+* Set your DNS Time-to-Live as low as possible.
+
+### 1. Create and populate a new project
+
+Create a new Platform.sh project in the desired region.  You can initially create it as a Development project and change the plan size immediately before switching over or go ahead and use the desired size from the beginning.  When the system asks if you want to use an existing template or provide your own code, select provide your own code.
+
+Make a Git clone of your existing project.  Then add a Git remote to the new project, using the Git URL shown in the UI.  Push the code for at least your master branch to the new project.  (You can also transfer other branches if desired.  That's optional.)
+
+Copy your existing user files on the old project to your computer using `rsync`.  See the [exporting](/tutorials/exporting.md) page for details.  Then use `rsync` to copy them to the same directory on the new project.  See the [migrating](/tutorials/migrating.md) page for details.
+
+Export your database from the old project and import it into the new project.  Again, see the exporting and migration pages, as well as the instructions for your specific database services.
+
+Re-enter any project or environment variables you've defined on your old project in your new project.
+
+Add any users to your new project that you want to continue to have access.
```

Comment:
Shall we remind customer to create Integrations on new project as well, especially the disk usage notification?

Reply:
Good call!  I'll add.